eos/observable_TEST.cc

```diff
@@ -127,6 +131,60 @@ class ObservableTest :
                     )")
                 );
             }
+
+            /* Test the names of the kinematic variables against a whitelist */
+            {
+                static const std::set<std::string> allowed_kinematic_variables
+                {
+                    "q2", "q2_min", "q2_max",
```

Comment:
Shouldn't we better write a list without min and max, and try to parse min and max in the test?

Reply:
Could be done. Does it help? Currently, not all variables follow the scheme ``"common_name" + "_min"``, e.g., ``w_min_denom``, which probably should be ``w_denom_min``.

We can adapt the test case once all offending instances of kinematic variable names have been removed?

Reply:
Sounds good